autoload/lsp/utils.vim

```diff
@@ -88,3 +88,9 @@ endif
 function! lsp#utils#empty_complete(...) abort
     return []
 endfunction
+
+function! s:error_msg(msg) abort
```

Comment:
need to change this to `lsp#utils#error(msg)`

Reply:
Done